lib/perl/Genome/Model/PhenotypeCorrelation/Command/CaseControl/Unrelated.pm

```diff
@@ -297,7 +297,8 @@ sub _create_workflow {
                 work_dir => $vep_work_directory,
                 ensembl_annotation_build => $ensembl_annotation_build,
                 log_dir => $log_dir,
-
+                analysis_build => $self->build,
+                analysis_project => $self->build->model->analysis_projects,
```

Comment:
I believe the `is_many` marking was a limitation of UR? I could be mistaken, but models have a bridge table to analysis projects (rather than a direct FK on the model table) and I thought at the time that UR could only handle such a relationship as `is_many` but I could be very wrong on that! If I'm wrong, I agree we should change it.